examples/quickstart.ipynb

```diff
@@ -105,7 +105,7 @@
    "source": [
     "# This may take a few minutes to download\n",
     "import urllib.request\n",
-    "s3_path = \"https://modin-datasets.s3.amazonaws.com/testing/yellow_tripdata_2015-01.csv\"\n",
+    "s3_path = \"https://modin-datasets.intel.com/testing/yellow_tripdata_2015-01.csv\"\n",
```

Comment:
```suggestion
    "url_path = \"https://modin-datasets.intel.com/testing/yellow_tripdata_2015-01.csv\"\n",
```
Please replace this var everywhere in this file.

Reply:
`url` is actually a path already, I'd better use `dataset_url`

Reply:
Fixed